specification/containerregistry/data-plane/Azure.ContainerRegistry/preview/2019-08-15-preview/containerregistry.json

```diff
@@ -1,3091 +1,3091 @@
-        },
-        "kid": {
-          "type": "string",
-          "description": "kid value"
-        },
-        "kty": {
-          "type": "string",
-          "description": "kty value"
-        },
-        "x": {
-          "type": "string",
-          "description": "x value"
-        },
-        "y": {
-          "type": "string",
-          "description": "y value"
-        }
-      }
-    },
-    "History": {
-      "description": "A list of unstructured historical data for v1 compatibility",
-      "x-accessibility": "internal",
-      "properties": {
-        "v1Compatibility": {
-          "type": "string",
-          "description": "The raw v1 compatibility information"
-        }
+          "allOf": [
+              {
+                  "$ref": "#/definitions/Manifest"
+              }
+          ]
       },
-      "example": {
-        "v1Compatibility": "v1 compatibility info"
-      }
-    },
-    "Repositories": {
-      "description": "List of repositories",
-      "properties": {
-        "repositories": {
-          "type": "array",
-          "description": "Repository names",
-          "items": {
-            "type": "string"
-          }
-        },
-        "link": {
-          "type": "string"
-        }
-      },
-      "x-accessibility": "internal",
-      "example": {
-        "repositories": [
-          "production/alpine",
-          "testing/alpine"
-        ]
-      }
-    },
-    "DeletedRepository": {
-      "x-ms-client-name": "DeleteRepositoryResult",
-      "description": "Deleted repository",
-      "properties": {
-        "manifestsDeleted": {
-          "type": "array",
-          "readOnly": true,
-          "description": "SHA of the deleted image",
-          "items": {
-            "type": "string"
+      "OCIIndex": {
+          "x-ms-discriminator-value": "application/vnd.oci.image.index.v1+json",
+          "x-accessibility": "internal",
+          "description": "Returns the requested OCI index file",
+          "properties": {
+              "manifests": {
+                  "type": "array",
+                  "description": "List of OCI image layer information",
+                  "items": {
+                      "$ref": "#/definitions/ManifestListAttributes"
+                  }
+              },
+              "annotations": {
+                  "$ref": "#/definitions/Annotations"
+              }
           },
-          "x-ms-client-name": "deletedManifests"
-        },
-        "tagsDeleted": {
-          "type": "array",
-          "readOnly": true,
-          "description": "Tag of the deleted image",
-          "items": {
-            "type": "string"
+          "allOf": [
+              {
+                  "$ref": "#/definitions/Manifest"
+              }
+          ]
+      },
+      "V1Manifest": {
+          "description": "Returns the requested V1 manifest file",
+          "x-ms-discriminator-value": "application/vnd.docker.distribution.manifest.v1+json",
```

Comment:
I believe this is a bug -- can you confirm?

Reply:
Oh yes, good catch